.vscode/launch.json

```diff
@@ -2,11 +2,43 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "Attach to Node Functions",
+            "name": "Debug API",
+            "type": "node",
+            "request": "launch",
+            "program": "${workspaceRoot}/api/node_modules/jest/bin/jest",
+            "args": [
+                "-i",
+                "--config",
+                "jest.config.js"
+            ],
+            "internalConsoleOptions": "openOnSessionStart",
+            "console": "integratedTerminal",
+            "outFiles": [
+                "${workspaceRoot}/api/build/dist/**/*"
+            ],
+            "cwd": "${workspaceFolder}/api"
+        },
+        {
+            "name": "Attach to Functions",
             "type": "node",
             "request": "attach",
             "port": 9229,
-            "preLaunchTask": "func: host start"
+            "preLaunchTask": "func: host start",
+            "cwd": "${workspaceFolder}/api"
+        },
+        {
+            "type": "chrome",
```

Comment:
this supports chromium edge now 😉

Reply:
Should I change this to edge? I am fine either way as I have both installed.

Reply:
(id test it if it works on your machine first. i remember triaging a bug with them a while back when it was first added since it couldn't actually locate the edge instance)

Reply:
done. and added the debug in edge extension to the recommended list for the project too